src/display/editor/ink.js

```diff
@@ -159,6 +165,19 @@ class InkEditor extends AnnotationEditor {
     return this.paths.length === 0;
   }
 
+  #getInitialBBox() {
+    const { width, height, rotation } = this.parent.viewport;
+    switch (rotation) {
+      case 90:
+        return [0, width, width, height];
+      case 180:
+        return [width, height, width, height];
+      case 270:
+        return [height, 0, width, height];
+    }
+    return [0, 0, width, height];
```

Comment:
In other `switch`-statements, for example in `screenToPageTranslation`, you've used the following format:
```suggestion
      case 270:
        return [height, 0, width, height];
      default:
        return [0, 0, width, height];
```